tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "include": ["**/**.js", "*.js"],
+  "exclude": ["coverage", "node_modules"],
```

```suggestion
  "exclude": ["coverage/", "node_modules/"],
```
Explicit that they are folders